math/FunMath.h

```diff
@@ -18,5 +24,12 @@
 */
 // int Sum(int num1, int num2);
 
+std::vector<int> EvenOddVectors(std::vector<int> even_odd_input) {
```

Comment:
Move your implementation into FunMath.cpp

You'll need to add a function comment that explains what this function does, what its parameters are, and what the return value is.